numba/unicode.py

```diff
@@ -745,6 +746,45 @@ def concat_impl(a, b):
         return concat_impl
 
 
+@njit
```

Comment:
If you have time and inclination, I'd suggest using the `@register_jitable` decorator as opposed to `@njit`. This is just a minor implementation detail, but it is the preferred way to make ancillary functions available to other jitted functions.

Reply:
No problem, I need to commit another flake8 fix, so I can add this. Can you tell me a bit more about this `@register_jitable`? Should I be usually using it in my code, or is it only for numba internal code?

Reply:
So, `register_jitable` isn't very well documented yet, but it calls `overload` under the hood. So, as far as I understand, basically it leaves your original Python function intact and the jitted version is only accessed  from other jitted functions . It is part of the `extending` API so I guess it was originally designed for extending Numba. However, I would assume that the decorator will remain stable, so in principle, there is nothing preventing you from using it. But maybe @stuartarchibald or @sklam can weight in on this too?